MonoGame.Framework/Platform/Media/Song.iOS.cs

```diff
@@ -193,15 +192,6 @@ private Genre PlatformGetGenre()
             return this.genre;
         }
 
-        private TimeSpan PlatformGetDuration()
```

Comment:
@mrhelmut i fixed these cases so it always assigns to the common _duration type.